data/build.gradle

```diff
@@ -1,6 +1,45 @@
 apply from: '../buildsystem/android_library.gradle'
 apply from: '../buildsystem/test_dependencies.gradle'
+apply from: '../buildsystem/test_android_dependencies.gradle'
+apply plugin: 'kotlin-kapt'
+
+android {
+    defaultConfig {
```

Comment:
maybe :contracts is more suitable for these constants, as they relate to contracts

Reply:
I tried but the `:contracts` module is a "java-library", I would have to change it to an android one. Are you ok with that?
Also, if we need to access this constants from :app, the :data module will always have to act as mediator between :app and :contracts. If you are fine with that I will push the change.

Reply:
I'm fine with the way it is right now. Having those constants directly in :app would also be good